standalone/src/main/java/com/criteo/publisher/samples/standalone/ContentItem.java

```diff
@@ -0,0 +1,15 @@
+package com.criteo.publisher.samples.standalone;
+
+public class ContentItem {
```

Comment:
So-called value object classes, such as this one, should be immutable. Immutability helps with thread safety and debugging as the value of an object doesn't change anymore after its creation (more info: https://dev.to/fragmented/075-effective-java--item-15). 
To achieve immutability, you need to make all attributes final.

```
public class ContentItem {
   private final String title;
   private final String description;
   private final String imageUrl;
  // constructor
}
```

Also, fields should be private.

Reply:
Done, thanks!